includes/Classifai/Blocks/recommended-content-block/inspector-controls/taxonomy-controls.js

```diff
@@ -4,12 +4,14 @@
  *
  * @see https://github.com/WordPress/gutenberg/blob/trunk/packages/block-library/src/query/edit/inspector-controls/taxonomy-controls.js
  */
-import { FormTokenField } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { FormTokenField, ToggleControl } from '@wordpress/components';
 import { useSelect } from '@wordpress/data';
 import { store as coreStore } from '@wordpress/core-data';
 import { getEntitiesInfo, useTaxonomies } from '../utils';
 
 const termsPerPage = 100;
+export const syncTaxonomies = [ 'category', 'post_tag' ];
```

Comment:
I know the original issue mentioned just categories and tags but I think we allow setting any taxonomies that are assigned to the current post type. Ideal would be to not hardcode these values but support all taxonomies the current post type supports